activiti-core-common-dependencies/pom.xml

```diff
@@ -65,6 +65,16 @@
         <artifactId>activiti-spring-security-policies</artifactId>
         <version>${activiti-core-common.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.activiti.core.common</groupId>
+        <artifactId>activiti-steps-assertions-api</artifactId>
```

Comment:
@salaboy shall we add `test` in the artifact name to make it explicit that this should be used for test purpose only, or assertions is it enough?

Reply:
I think that we should be as explicit as possible .. activiti-core-api-test (and activiti-core-impl-test) should be ok as a module name.. the fact that it has assertions is just a consequence of being a test lib..

Reply:
Yes, it makes sense, but actually, I'm not sure about having `api` in the name as it's providing more than interfaces: all the assertions are in there. Maybe `activiti-core-test-assertions` (instead of `activiti-steps-assertions-api`) and `activiti-core-test-local-runtime` (instead of  `activiti-steps-assertions`) would be better names for these modules. What do you think?

Reply:
👍 yes.. that make sense.